js/lykke2.js

```diff
@@ -0,0 +1,1171 @@
+        //     ]
+        //
+        const result = { 'info': response };
+        for (let i = 0; i < response.length; i++) {
+            const balance = response[i];
+            const currencyId = this.safeString (balance, 'assetId');
+            const code = this.safeCurrencyCode (currencyId);
+            const account = this.account ();
+            const free = this.safeString (balance, 'available');
+            const used = this.safeString (balance, 'reserved');
+            account['free'] = free;
+            account['used'] = used;
+            result[code] = account;
+        }
+        return this.safeBalance (result);
+    }
+
+    async fetchBalance (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetBalance (params);
+        const payload = this.safeValue (response, 'payload', []);
+        //
+        //     {
+        //         "payload":[
+        //             {
+        //                 "assetId":"2a34d6a6-5839-40e5-836f-c1178fa09b89",
+        //                 "available":0.1,
+        //                 "reserved":0.0,
+        //                 "timestamp":1644146723620
+        //             }
+        //         ],
+        //         "error":null
+        //     }
+        //
+        return this.parseBalance (payload);
+    }
+
+    parseOrderStatus (status) {
+        const statuses = {
+            'Open': 'open',
+            'Pending': 'open',
+            'InOrderBook': 'open',
+            'Processing': 'open',
+            'Matched': 'closed',
+            'Cancelled': 'canceled',
+            'Rejected': 'rejected',
+            'Replaced': 'canceled',
+            'Placed': 'open',
+        };
+        return this.safeString (statuses, status, status);
+    }
+
+    parseOrder (order, market = undefined) {
+        //
+        //     {
+        //         "id":"1b367978-7e4f-454b-b870-64040d484443",
+        //         "timestamp":1644155923357,
+        //         "lastTradeTimestamp":1644155923357,
+        //         "status":"Matched",
+        //         "assetPairId":"BCHEUR",
+        //         "type":"Market",
+        //         "side":"Sell",
+        //         "price":280.569,
+        //         "volume":0.01,
+        //         "filledVolume":0.01,
+        //         "remainingVolume":0.0,
+        //         "cost":2.80569
+        //     }
+        //
+        const id = this.safeString (order, 'id');
+        const status = this.parseOrderStatus (this.safeString (order, 'status'));
+        const marketId = this.safeString (order, 'assetPairId');
+        const symbol = this.safeSymbol (marketId, market);
+        const type = this.safeString (order, 'type');
+        const lastTradeTimestamp = this.safeInteger (order, 'lastTradeTimestamp');
+        const timestamp = this.safeInteger (order, 'timestamp');
+        const price = this.safeString (order, 'price');
+        const side = this.safeString (order, 'side');
```

Comment:
we need to use `safeStringLower` here fore the side, since the first letter is capitalised and ccxt expects a lower case string "Sell" vs "sell"

Reply:
same thing with `type`